Package.swift

```diff
@@ -10,23 +10,22 @@ let package = Package(
         // Products define the executables and libraries a package produces, and make them visible to other packages.
         .library(
             name: "NostrSDK",
-            targets: ["NostrSDK"]),
+            targets: ["NostrSDK"])
     ],
     dependencies: [
         // Dependencies declare other packages that this package depends on.
         // .package(url: /* package url */, from: "1.0.0"),
-        .package(url: "https://github.com/realm/SwiftLint.git", .upToNextMinor(from: "0.52.2"))
```

Comment:
I guess it makes sense that we don't want to necessarily pull in this dependency as part of what's required to use this library. We'll just have to run SwiftLint locally in a different way -- but I suppose the build at PR time should catch lint issues.